ui/lib/utils/sqlFormatter/index.ts

```diff
@@ -1,17 +1,9 @@
-// import TiDBSQLFormatter from './TiDBSQLFormatter'
-
-// const mySqlFormatter = new TiDBSQLFormatter({ uppercase: true })
-
-// export default function formatSql(sql?: string): string {
-//   return mySqlFormatter.format(sql || '')
-// }
-
-import { format } from 'sql-formatter'
+import { format } from '@baurine/sql-formatter-plus'
```

Comment:
We should ensure that tests are contained in order to avoid regression.

Reply:
ok, I will add them back in the dashboard. I added them in the `@baurine/sql-formatter-plus` as well.